Schema guide descriptions for root-level keys `preferred-citation` and `references` (#250)
citation-file-format/citation-file-format: schema-guide.md

- **type**: A [`definitions.reference`](#definitionsreference) object.
- **required**: `false`

> - **description**: The reference that should be cited when citing the work, instead of using the software or dataset metadata in this CITATION.cff file.  

Author: ```suggestion
- **description**: A reference to another work that should be cited instead of the software or dataset itself.  
```

Changed according to the proposed change in the schema.
